pkg/common/maa.go

```diff
@@ -69,7 +68,7 @@ func newAttestSNPRequestBody(snpAttestationReport []byte, vcekCertChain []byte,
 	}
 	logrus.Debugf("base64urlEncodedUvmReferenceInfo: %s", base64urlEncodedUvmReferenceInfo)
 
-	if common.GenerateTestData {
+	if GenerateTestData {

```

Comment:
would it be better to make this an env var instead of an global shared variable?

Reply:
future work will resolve this